assets/js/vue/components/search/ResultSet.vue

```diff
@@ -11,6 +11,13 @@
         </section>
         <section class="section-content bg padding-y">
             <div class="container">
+                <b-pagination
+                        v-model="currentPage"
+                        :total-rows="rows"
+                        :per-page="perPage"
+                        aria-controls="my-table"
```

Comment:
Is this specifically needed?

Reply:
No, copy pasted code from bootstrap documentation page